ChangeLog

```diff
@@ -34,6 +34,8 @@ Features
      The following functions from the ECDH module can be replaced
      with an alternative implementation:
      mbedtls_ecdh_gen_public() and mbedtls_ecdh_compute_shared().
+   * Add support for alternative implementation for ECJPAKE, controlled by
+     new configuration flag MBEDTLS_ECJPAKE_ALT.
```

Comment:
We need an article before `new`.

```diff
- new configuration flag MBEDTLS_ECJPAKE_ALT.
+ the new configuration flag MBEDTLS_ECJPAKE_ALT.
```

Reply:
@Patater Thanks - I'll amend the commit if you're ok with it.

Reply:
Yeah, I'm OK with that.